stable/pipelines/README.md

```diff
@@ -216,6 +187,18 @@ To start using Pipelines you need to setup a Build Plane:
 
 - For Kubernetes Node-pool setup, please read [Managing Dynamic Node Pools](https://www.jfrog.com/confluence/display/JFROG/Managing+Pipelines+Node+Pools#ManagingPipelinesNodePools-dynamic-node-poolsAdministeringDynamicNodePools).
 
+### Using Vault in Production environments
+To use vault securely you must set the disablemlock setting in the values.yaml to false as per the Hashicorp Vault recommendations here:
+
+https://www.vaultproject.io/docs/configuration#disable_mlock
+
+For dev or test environment it is acceptable to leave this value set to true.
+
+Note however this does enable a potential security issue where encrypted credentials could potentially be swapped onto an unencrypted disk. 
+
+For this reason we recommend you always set this value to false to ensure mlock is enabled.
+
+
```

Comment:
Please add an example from the values.yaml how to set to `true` or false`

Reply:
Added to values.yaml that you can set it to true for dev or test and false for production grade systems.